Microsoft.Toolkit.Uwp.SampleApp/SamplePages/StringExtensions/StringExtensionsPage.xaml

```diff
@@ -9,9 +9,13 @@
     mc:Ignorable="d">
 
     <Grid>
-        <StackPanel Orientation="Vertical" VerticalAlignment="Center" HorizontalAlignment="Center">
-            <TextBox PlaceholderText="Enter String" HorizontalAlignment="Center" VerticalAlignment="Center" x:Name="InputTextBox" MinWidth="300" />
-            <TextBlock x:Name="IsValid" Padding="0,5"/>
+        <StackPanel Orientation="Vertical" VerticalAlignment="Center" HorizontalAlignment="Center" Margin="10">
+            <TextBox PlaceholderText="Enter String" HorizontalAlignment="Center" VerticalAlignment="Center" x:Name="InputTextBox" MinWidth="300" TextChanged="OnTextChanged" />
+            <TextBlock x:Name="IsValidEmail" Padding="0,5"/>
```

Comment:
format the layout of the content into a grid with two columns, keep the first column as a constant
```
<TextBlock Text="IsEmail():" Margin="0,0,6,0"/>
<TextBlock x:Name="IsValidEmail Grid.Column="1"/>
```

Reply:
Just saw the comments above. Note that this is only a comment as well, not a requested change. I think the layout makes a big difference in readability

Reply:
If it was in a grid, would you think right-aligned labels, or left-aligned?

Another alternative would be to make the result (false or true) bold.

Reply:
IMO, I think the result should be using the boolean color guidelines in Visual Studio: both bold and blue.

Reply:
> I think the result should be using the boolean color guidelines in Visual Studio: both bold and blue.

Where are there guidelines?
Were they used to guide other decisions on the UI of the sample app?

Reply:
No. There is no guidelines for sample app as I know. I meant that we can use design of Visual Studio to highlight true/false value. But of course, bold only is also a perfect solution.

Reply:
Right aligned labels

Reply:
How about two columns, left column content right aligned, right column content left aligned and bold with a small margin in between? This way things will be centered and still aligned.

Reply:
Will this make everyone happy?

![image](https://user-images.githubusercontent.com/189547/40059096-0d956268-584b-11e8-9064-f5391d914bdb.png)